store/mongo/datastore_mongo.go

```diff
@@ -595,3 +595,125 @@ func (db *DataStoreMongo) GetLimit(ctx context.Context, name string) (*model.Lim
 
 	return &lim, nil
 }
+
+func (db *DataStoreMongo) GetDevCountByStatus(ctx context.Context, status string) (int, error) {
```

Comment:
this looks a bit complicated, maybe I'm not getting full intention but that was supposed to count devices of specified status, devices already have status field so there is no need to mine authsets (is there?)

then something like this ```db.devices.find({"status":"accepted"}).count()``` could do the trick

Reply:
the thing is the `device.status` field is obsolete and not consistently updated since authsets were introduced. the idea I got from talking to the rest of the team is that we will drop it at some point.

so auth sets are the actual 'source of truth' on the overall status.

the query is complicated because I did too much - I didn't notice that only `accepted` devices must be counted for now, and automatically assumed we're after all statuses. it was pointed out to me later, and since I already started composing the query I didn't want to drop it - I imagine at some point counting other statuses will be useful.

Reply:
I see, thanks for explanation.